go.mod

```diff
@@ -1,4 +1,4 @@
-module kubevirt.io/node-maintenance-operator
+module medik8s.io/node-maintenance-operator
```

Comment:
this won't work if anyone wants import our code, because medik8s.io/node-maintenance-operator can't be resolved. Please use github.com/medik8s/node-maintenance-operator. And then update all imports accordingly.